glide.yaml

```diff
@@ -3,7 +3,7 @@ import:
 - package: go.uber.org/multierr
   version: ^1
 - package: go.uber.org/dig
-  version: ^1.4
+  version: ^1.5
```

Comment:
We've been bad about this in the past; please leave a comment noting that we need at least v1.5 for the `DeferAcyclicVerification` option.